src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -598,7 +598,12 @@
     <Copy SourceFiles="@(PackageVersionPropsSnapshotFiles)" DestinationFolder="$(PackageReportDir)snapshots/" />
 
     <ItemGroup>
-      <RestoredPackageFile Include="$(NuGetPackageRoot)**/*.nupkg"
+      <!--
+        Note the usage of */*/*.nupkg and not **/*.nupkg. The latter would include nupkgs
+        carried as content, i.e. in Microsoft.FSharp.Compiler nupkg, and cause false positives.
+        Content nupkgs were not restored and should not be used in prebuit detection.
+      -->
+      <RestoredPackageFile Include="$(NuGetPackageRoot)*/*/*.nupkg"
```

Comment:
Please add an example of a path in the above description to better understand the glob.

Reply:
I.e. something like `<packageid>/<packageversion>/...`

Reply:
Will update - thanks!

Reply:
Fixed with https://github.com/dotnet/installer/pull/19404/commits/9744c072159cfb4f9743ba9319c66939e42549ec